docs/tutorials/control/gamepad.md

```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
+For more information about the controller buttons and Bluetooth modes, consult the manual included with the controller.
+
+Run `sudo bluetoothctl scan on` to list all Bluetooth devices within reach of your machine.
+As you do this, make sure you run this command on the machine that runs `viam-server` which may be a different machine to your laptop.
+
+This command scans all bluetooth devices.
+Find your gamepad.
+For an 8bitdo controller, the MAC address begins with E4:17:D8.
+
+![A screenshot of a Raspberry Pi terminal with the following command: sudo bluetoothctl scan on. The results of the command are displayed: a list of device MAC addresses.](/tutorials/yahboom-rover/bluetooth-scan.png)
+
+Once you find your controller, pair with the controller by running the following command: `sudo bluetoothctl pair <8bitdo-mac-address>`.
+Do not forget to take the < and > symbols out as you paste your address.
+
+Then connect the controller: `sudo bluetoothctl connect <8bitdo-mac-address>`
+
+To make reconnecting easier in the future, trust the controller by running the following command: `sudo bluetoothctl trust <8bitdo-mac-address>`
+
+To confirm the connection, you can list connected devices with: `sudo bluetoothctl devices | cut -f2 -d| while read uuid; do sudo bluetoothctl info $uuid; done|grep -e "Device\|Connected\|Name"`
+
+![A screenshot of a Pi terminal showing the above bluetoothctl commands and their outputs.](/tutorials/yahboom-rover/bluetoothpair-connect.png)
+
+If you would like a stronger understanding of `bluetoothctl` and managing Bluetooth devices in Linux, we recommend [How to Manage Bluetooth Devices on Linux Using bluetoothctl](https://www.makeuseof.com/manage-bluetooth-linux-with-bluetoothctl/).
+
+{{% /expand%}}
 
 ## Add the controller to the rover's config
 
-Go to your rover's **Config** tab on the [Viam app](https://app.viam.com/).
+Go to your rover's **CONFIGURE** tab on the [Viam app](https://app.viam.com/).
 
 {{< tabs >}}
 {{% tab name="Config Builder" %}}
 
 Configure a [gamepad](/components/input-controller/gamepad/):
 
-- Click **Create component**.
-- Select `input_controller` for the component **Type**.
-- Select `gamepad` for the **Model**.
-- Enter `my-gamepad` as the component **Name**.
-- Click **Create**.
+Click the + icon next to Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `input_controller` type, then select the `gamepad` model.
+Enter a name or use the suggested name for your input controller and click **Create**.
 
-![Blank configuration JSON](/tutorials/scuttle-gamepad/gamepad-config.png)
+![An example configuration for a linux-based gamepad input controller component in the Viam App config builder](/components/input-controller/gamepad-input-controller-ui-config.png)
+
+You can set the `auto_reconnect` attribute to `true`.
 
 {{% /tab %}}
 {{% tab name="Raw JSON" %}}
```

Comment:
```suggestion
{{% tab name="JSON" %}}
```
Have been switching this as subtab name is just JSON now. Doesn't seem like a big deal either way though. Was considering also doing Config Builder - Builder but haven't done that yet, any thoughts?

Reply:
Sounds good I think